specification/datadog/resource-manager/Microsoft.Datadog/stable/2023-01-01/datadog.json

```diff
@@ -0,0 +1,2503 @@
+    "LinkedResourceListResponse": {
+      "description": "Response of a list operation.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Results of a list operation.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LinkedResource"
+          }
+        },
+        "nextLink": {
+          "description": "Link to the next set of results, if any.",
+          "type": "string"
+        }
+      }
+    },
+    "MonitoredResource": {
+      "description": "The properties of a resource currently being monitored by the Datadog monitor resource.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The ARM id of the resource.",
+          "type": "string"
+        },
+        "sendingMetrics": {
+          "description": "Flag indicating if resource is sending metrics to Datadog.",
+          "type": "boolean"
+        },
+        "reasonForMetricsStatus": {
+          "description": "Reason for why the resource is sending metrics (or why it is not sending).",
+          "type": "string"
+        },
+        "sendingLogs": {
+          "description": "Flag indicating if resource is sending logs to Datadog.",
+          "type": "boolean"
+        },
+        "reasonForLogsStatus": {
+          "description": "Reason for why the resource is sending logs (or why it is not sending).",
+          "type": "string"
+        }
+      }
+    },
+    "MonitoredResourceListResponse": {
+      "description": "Response of a list operation.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Results of a list operation.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MonitoredResource"
+          }
+        },
+        "nextLink": {
+          "description": "Link to the next set of results, if any.",
+          "type": "string"
+        }
+      }
+    },
+    "MonitoredSubscription": {
+      "description": "The list of subscriptions and it's monitoring status by current Datadog monitor.",
+      "type": "object",
+      "properties": {
+        "subscriptionId": {
+          "description": "The subscriptionId to be monitored.",
+          "required": [
+            "subscriptionId"
+          ],
+          "type": "string"
+        },
+        "status": {
+          "description": "The state of monitoring.",
+          "enum": [
+            "InProgress",
+            "Active",
+            "Failed",
+            "Deleting"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "Status"
+          },
+          "type": "string"
+        },
+        "error": {
+          "description": "The reason of not monitoring the subscription.",
+          "type": "string"
+        },
+        "tagRules": {
+          "$ref": "#/definitions/MonitoringTagRulesProperties"
+        }
+      }
+    },
+    "SubscriptionList": {
```

Comment:
>SubscriptionList [](http://example.com/codeflow?start=5&length=16)

can we rename it MonitoredSubscriptionsList to make it more intuitive?

Reply:
Point taken. Since this will be a breaking change, we'll raise a separate PR just to accommodate all the changes mentioned by you. Currently, the purpose of the PR is to accommodate multi-subscription changes.